llvm/lib/Transforms/Scalar/LowerMatrixIntrinsics.cpp

```diff
@@ -440,6 +442,10 @@ class LowerMatrixIntrinsics {
       return make_range(Vectors.begin(), Vectors.end());
     }
 
+    iterator_range<SmallVector<Value *, 8>::const_iterator> vectors() const {
```

Comment:
Are the non-const variants of the iterator actually needed or can all current users be served by the `const` one?